api/app/database.py

```diff
@@ -27,7 +29,22 @@ def get_user(user_id):
     try:
         conn = _connect()
         with conn, conn.cursor() as cur:
-            cur.execute("SELECT * FROM users WHERE user_id = %s", (user_id,))
-            return cur.fetchone()
+            if user_id == "all":
```

Comment:
Was this left here accidentally during debugging, or is it intentional to allow GET to do this?

Reply:
I intended for it to be an extra way to show a list of all users since it's hard to obtain the user id manually.